custom_components/bmw_connected_drive/sensor.py

```diff
@@ -28,10 +32,17 @@
     "remaining_range_fuel": ["mdi:map-marker-distance", LENGTH_KILOMETERS],
     "max_range_electric": ["mdi:map-marker-distance", LENGTH_KILOMETERS],
     "remaining_fuel": ["mdi:gas-station", VOLUME_LITERS],
-    "charging_time_remaining": ["mdi:update", TIME_HOURS],
-    "charging_status": ["mdi:battery-charging", None],
-    # No icon as this is dealt with directly as a special case in icon()
-    "charging_level_hv": [None, PERCENTAGE],
+    # LastTrip attributes
+    "avgCombinedConsumption": ["mdi:flash", f"{ENERGY_KILO_WATT_HOUR}/100{LENGTH_KILOMETERS}"],
```

Comment:
In HA they're not in CamelCase. The attributes in the BimmerConnected JSON have these names, so to the reference to these fields are the same. As long a the service attributes are not available in any method like 'available_attributes', I'm referring directly to the JSON-fields...

Reply:
These names are used for the entity_ids. We always use `snake_case` in HA for this and also should be using that as much as possible in the library, so please keep it uniform.

https://github.com/bimmerconnected/bimmer_connected/blob/dev/bimmer_connected/last_trip.py#L109

In HA it's now `snake_case` and `CamelCase` combined which is not how we should do this.

![Screenshot_2](https://user-images.githubusercontent.com/11230573/98915098-be9e3d80-24c9-11eb-9065-e498ce54536c.png)

Reply:
Resolved by looping through available attributes instead of looping through statistics-JSON. 
btw, imho there should be an available_attributes in BimmerConnected right?